--- plt/collects/make/setup-extension.ss.orig	2004-01-03 06:26:15.000000000 +0100
+++ plt/collects/make/setup-extension.ss	2005-05-11 19:47:50.000000000 +0200
@@ -94,7 +94,7 @@
 				(ormap (lambda (suffix)
 					 (file-exists? 
 					  (build-path x 
-						      "lib"
+						      "lib64"
 						      (format "~a~a.~a" 
 							      (if is-win? 
 								  ""
@@ -137,7 +137,7 @@
 	  current-extension-linker-flags 
 	  (if is-win?
 	      null
-	      (list (format "-L~a/lib" sys-path)))
+	      (list (format "-L~a/lib64" sys-path)))
 	  
 	  ;; Add libs for Windows:
 	  (with-new-flags
--- plt/src/Makefile.in.orig	2004-05-20 05:00:01.000000000 +0200
+++ plt/src/Makefile.in	2005-05-11 20:23:20.000000000 +0200
@@ -73,7 +73,7 @@
 
 
 lib-finish:
-	@LIBFINISH@ @prefix@/lib
+	@LIBFINISH@ @prefix@/lib64
 
 srcdir = @srcdir@
 prefix = @prefix@
--- plt/src/mred/Makefile.in.orig	2004-06-29 10:20:52.000000000 +0200
+++ plt/src/mred/Makefile.in	2005-05-11 20:39:02.000000000 +0200
@@ -225,7 +225,7 @@
 	echo "no dynamic libs"
 
 install-lib:
-	cd ..; $(ICP) mred/libmred.@LIBSFX@ `(cd $(prefix); pwd)`/lib/
+	cd ..; $(ICP) mred/libmred.@LIBSFX@ `(cd $(prefix); pwd)`/lib64/
 
 install_wx_xt:
 	cd ..; if [ ! -d $(prefix)/bin ] ; then mkdir $(prefix)/bin ; fi
@@ -253,7 +253,7 @@
 	ln -s Versions/$(FWVERSION)/Resources $(MRFWDIR)/
 	cd ..; $(ICP) -r mred/Starter.app $(prefix)/collects/launcher/.
 
-LIBIDIR=$(prefix)/lib
+LIBIDIR=$(prefix)/lib64
 BUILDINFO=$(LIBIDIR)/buildinfo
 
 install:
--- plt/src/mzscheme/Makefile.in.orig	2004-07-21 10:30:03.000000000 +0200
+++ plt/src/mzscheme/Makefile.in	2005-05-11 20:40:10.000000000 +0200
@@ -235,12 +235,12 @@
 install-3m-basic:
 	cd ..; rm -f $(prefix)/bin/mzscheme3m
 	cd ..; cp mzscheme/mzscheme3m $(prefix)/bin/mzscheme3m
-	cd ..; cp mzscheme/mzdyn3m.o $(prefix)/lib/mzdyn3m.o
+	cd ..; cp mzscheme/mzdyn3m.o $(prefix)/lib64/mzdyn3m.o
 
 install-3m@NOT_OSX@:
 	$(MAKE) install-3m-basic
 
-LIBIDIR=`(cd $(prefix); pwd)`/lib
+LIBIDIR=`(cd $(prefix); pwd)`/lib64
 BUILDINFO=$(LIBIDIR)/buildinfo
 ICP=@ICP@
 
@@ -283,7 +283,7 @@
 	if [ ! -d $(MZFWDIR)/Versions/$(FWVERSION)_3m ] ; then mkdir $(MZFWDIR)/Versions/$(FWVERSION)_3m ; fi
 	cp $(MZFWMMM) $(MZFWDIR)/Versions/$(FWVERSION)_3m/
 
-WLIBIDIR=`(cd $(prefix); pwd)`/lib
+WLIBIDIR=`(cd $(prefix); pwd)`/lib64
 WBUILDINFO=$(WLIBIDIR)/buildinfo
 
 cygwin-install:
--- plt/src/mzscheme/configure.in.orig	2004-07-22 10:21:19.000000000 +0200
+++ plt/src/mzscheme/configure.in	2005-05-11 21:41:30.000000000 +0200
@@ -122,7 +122,7 @@
     echo "----> Installation will go to ${prefix}:"
     echo "---->   ${prefix}/collects/..."
     echo "---->   ${prefix}/bin/..."
-    echo "---->   ${prefix}/lib/..."
+    echo "---->   ${prefix}/lib64/..."
     echo "---->   ${prefix}/include/..."
     echo "---->   ${prefix}/man/..."
     echo "---->   ${prefix}/notes/..."
@@ -806,12 +806,12 @@
 
   plt_lib_version=`grep " MZSCHEME_VERSION " ${srcdir}/mzscheme/src/schvers.h | cut -d '"' -f 2`
 
-  AR="${LTDIR}/libtool --mode=link $CC${need_gcc_static_libgcc} -release ${plt_lib_version} -rpath ${absprefix}/lib -o"
+  AR="${LTDIR}/libtool --mode=link $CC${need_gcc_static_libgcc} -release ${plt_lib_version} -rpath ${absprefix}/lib64 -o"
   STATIC_AR="${LTDIR}/libtool --mode=link $CC -o"
   ARFLAGS=""
   RANLIB=":"
-  MZLINKER="${LTDIR}/libtool --mode=link $CC${need_gcc_static_libgcc} -rpath ${absprefix}/lib"
-  MREDLINKER="${LTDIR}/libtool --mode=link $CXX${need_gcc_static_libgcc} -rpath ${absprefix}/lib"
+  MZLINKER="${LTDIR}/libtool --mode=link $CC${need_gcc_static_libgcc} -rpath ${absprefix}/lib64"
+  MREDLINKER="${LTDIR}/libtool --mode=link $CXX${need_gcc_static_libgcc} -rpath ${absprefix}/lib64"
   PLAIN_CC="$CC"
   CC="${LTDIR}/libtool --mode=compile $CC"
   CXX="${LTDIR}/libtool --mode=compile $CXX"
--- plt/src/mzscheme/dynsrc/Makefile.in.orig	2005-05-11 21:46:30.000000000 +0200
+++ plt/src/mzscheme/dynsrc/Makefile.in	2005-05-11 21:55:04.000000000 +0200
@@ -47,7 +47,7 @@
 oe.o: $(srcdir)/oe.c $(HEADERS)
 	$(PLAIN_CC) $(CFLAGS) -fPIC -c $(srcdir)/oe.c -o oe.o
 
-ILIBDIR = $(prefix)/lib
+ILIBDIR = $(prefix)/lib64
 
 # Prefix might be relative to srcdir, or it might be absolute, so we
 # have to go up and install things from there.
--- plt/collects/dynext/private/dirs.ss.orig	2001-08-28 22:08:29.000000000 +0200
+++ plt/collects/dynext/private/dirs.ss	2005-05-11 22:11:02.000000000 +0200
@@ -16,7 +16,7 @@
 		      (loop (cdr l)))))))))
 
   (define include-dir (find-dir "include"))
-  (define std-library-dir (find-dir "lib"))
+  (define std-library-dir (find-dir "lib64"))
 
   (provide include-dir std-library-dir))
 
