--- plt/src/mzscheme/configure.in.orig	2004-07-22 10:21:19.000000000 +0200
+++ plt/src/mzscheme/configure.in	2005-05-12 19:47:59.000000000 +0200
@@ -369,7 +369,7 @@
        fi
        ;;
       alpha)
-       EXTRA_GMP_OBJ="gmp_alpha_gcc.o"
+       EXTRA_GMP_OBJ="gmp_alpha_gcc.lo"
        ;;
       *)
        ;;
--- plt/src/mzscheme/src/Makefile.in.orig	2003-06-21 01:04:41.000000000 +0200
+++ plt/src/mzscheme/src/Makefile.in	2005-05-12 20:21:54.000000000 +0200
@@ -216,7 +216,7 @@
 	$(CC) $(CFLAGS) -c $(srcdir)/vector.c -o vector.@LTO@
 
 gmp_alpha_gcc.@LTO@: $(srcdir)/gmp/gmp_alpha_gcc.s
-	$(AS) -o gmp_alpha_gcc.@LTO@ $(srcdir)/gmp/gmp_alpha_gcc.s
+	$(CC) --tag=CC -o gmp_alpha_gcc.@LTO@ -c $(srcdir)/gmp/gmp_alpha_gcc.s
 
 SCONFIG =   $(srcdir)/../sconfig.h $(srcdir)/../uconfig.h
 
