--- plt/install~	Wed Aug 28 17:56:10 2002
+++ plt/install	Thu Oct 10 18:25:52 2002
@@ -7,19 +7,14 @@
 #  through the platform's standard path)
 # Creates .zo files if the user assents
 
-if [ ! \( \( -x install \) -a \( -d collects \) \) ] ; then
-  echo "install: must be run from its own directory"
-  exit 1
-fi
-
-exec ./bin/mzscheme -qr "$0" ${1+"$@"}
+exec mzscheme -qr "$0" ${1+"$@"}
 echo Couldn't start MzScheme --- install incomplete!
 exit
 
 |#
 
 (unless (equal? #() argv)
-  (error './install "no arguments allowed on the command line")
+  (error 'install "no arguments allowed on the command line")
   (exit 1))
 
 (define release-date "August 2002")
@@ -21,10 +21,8 @@
 
 (define didnothing " (nothing to do)")
 
-(define plthome (current-directory))
-(putenv "PLTHOME" plthome)
-(putenv "PLTCOLLECTS" "")
-(current-library-collection-paths (list (build-path plthome "collects")))
+(define plthome (getenv "PLTHOME"))
+(current-library-collection-paths (list (build-path plthome "share/drscheme/collects")))
 
 (define plthome (regexp-replace* "\"" plthome "\\\\\""))
 
