--- plt/src/mzscheme/dynsrc/Makefile.in.orig	2003-09-29 15:48:24.000000000 +0200
+++ plt/src/mzscheme/dynsrc/Makefile.in	2005-05-11 19:27:36.000000000 +0200
@@ -32,20 +32,20 @@
 MZDYNDEP = ../mzdyn.o $(srcdir)/../include/ext.exp $(srcdir)/../include/mzscheme.exp
 
 ../mzdyn.o: $(srcdir)/mzdyn.c $(srcdir)/../src/schvers.h  $(HEADERS)
-	$(PLAIN_CC) $(CFLAGS) -c $(srcdir)/mzdyn.c -o ../mzdyn.o
+	$(PLAIN_CC) $(CFLAGS) -fPIC -c $(srcdir)/mzdyn.c -o ../mzdyn.o
 
 ../mzdyn3m.o: $(srcdir)/mzdyn.c $(srcdir)/../src/schvers.h  $(HEADERS)
-	$(PLAIN_CC) $(CFLAGS) -DMZ_PRECISE_GC -c $(srcdir)/mzdyn.c -o ../mzdyn3m.o
+	$(PLAIN_CC) $(CFLAGS) -fPIC -DMZ_PRECISE_GC -c $(srcdir)/mzdyn.c -o ../mzdyn3m.o
 
 ../dynexmpl.so: dynexmpl.o $(MZDYNDEP)
 	$(DYNLINK) ../dynexmpl.so dynexmpl.o
 dynexmpl.o: $(srcdir)/dynexmpl.c  $(HEADERS)
-	$(PLAIN_CC) $(CFLAGS) -c $(srcdir)/dynexmpl.c -o dynexmpl.o
+	$(PLAIN_CC) $(CFLAGS) -fPIC -c $(srcdir)/dynexmpl.c -o dynexmpl.o
 
 ../oe.so: oe.o  $(MZDYNDEP)
 	$(DYNLINK) ../oe.so oe.o
 oe.o: $(srcdir)/oe.c $(HEADERS)
-	$(PLAIN_CC) $(CFLAGS) -c $(srcdir)/oe.c -o oe.o
+	$(PLAIN_CC) $(CFLAGS) -fPIC -c $(srcdir)/oe.c -o oe.o
 
 ILIBDIR = $(prefix)/lib
 
